django/db/models/base.py

```diff
@@ -1708,6 +1708,8 @@ def _check_ordering(cls):
             fld = None
             for part in field.split(LOOKUP_SEP):
                 try:
+                    if _cls is not None and part == 'pk':
+                        part = _cls._meta.pk.name
```

Comment:
I don't think you need to guard against `_cls is None` here and you can skip the lookup in this case.

```python
if part == 'pk':
    fld = _cls._meta.pk
else:
    fld = _cls._meta.get_field(part)
```

Coincidentally this is another case where allowing `get_field('pk')` to work would have avoided this issue. Maybe it's time to revisit #8191.

Reply:
@charettes Thanks. Done.

Reply:
@charettes Agreed, but #8191 have to wait for Django 3.1, so this fix looks sufficient for backporting.